chart/templates/deployment.yaml

```diff
@@ -78,6 +78,14 @@ spec:
                   fieldPath: metadata.namespace
             - name: SCRAPE_INTERVAL
               value: '{{ ((.Values.prometheus).scrapeInterval) | default "5s" }}'
+            - name: CREATE_TARGETS
+              value: '{{ .Values.prometheus.createTargets }}'
+            - name: CREATE_ALERTS
+              {{- if (.Values.prometheus).createTargets}}
+              value: '{{.Values.prometheus.createAlerts}}'
+              {{ else }}
+              value: 'false'
+              {{ end }}
```

Comment:
I would prefer the validation of `createAlerts` can only be true if `createTargets` is true to be dona via a json schema.
@pchila can you provide some help here?

Reply:
I think that can be done via the JSON schema validation, we have similar constructs in the job-executor-service [values.schema.json](https://github.com/keptn-contrib/job-executor-service/blob/main/chart/values.schema.json#L101-L155).